data/transport missions.txt

```diff
@@ -3469,3 +3469,114 @@ mission "Inheritance Redirection"
 				decline
 	on complete
 		dialog "There is no inheritance for Mr. Pappa waiting for you at the bank, but there are many other victims that paid some credits while expecting to earn a larger amount of credits in return. Some make a big fuss about it, until the alerted authorities start fining them for attempted inheritance fraud. You leave the bank in silence."
+
+
+
+mission "Hauler VI cargo prototype 1"
+	name "Escort Hauler prototype to formal dinner"
+	description "Escort the Hauler prototype to <destination> to deliver sushi to a formal dinner."
+	source
+		neighbor planet "Skymoot"
+		not planet "Skymoot"
+	destination "Skymoot"
+	npc accompany
+		government "Merchant"
+		personality escort timid
+		ship "Hauler III" "SB-Labs Testament"
+	on offer
+		conversation
+			`An elegant-looking lady approaches you.`
+			`	"Captain <last>, Captain <last>! It is such an honor to meet you!"`
+			choice
+				`	"Wow, that's quite a greeting. Why is it an honor to meet me?"`
+				`	"Hello, nice to meet you too."`
+					goto "hello too"
+				`	"Sorry, I don't have time to talk. I'm really busy."`
+					defer
+			`	She smiles and replies, "Well, I actually don't know you, but I just read your name on one of the cargo manifests. And my experience is that this greeting works best if you need something from a spaceship captain."`
+			label "hello too"
+			`She says, "My name is Vanessa Oxygenpocket and I am part of a team that is designing the new Hauler VI transport ship series here at Southbound Shipyards."`
```

Comment:
`I am part of a team that is` lack of contractions makes people sound like Borg drones

Reply:
Changing to `I'm` and `that's`